docs/en/Running-Hive-with-Alluxio.md

```diff
@@ -46,8 +46,8 @@ Hive's Hadoop jobs will store its intermediate results in Alluxio. Please follow
 
 ### Add additional Alluxio site properties to Hive
 
-If there are any Alluxio site properties you want to specify for Hive, add those to `hive-site.xml`. For example,
-change `alluxio.user.file.writetype.default` from default `MUST_CACHE` to `CACHE_THROUGH`:
+If there are any Alluxio site properties you want to specify for Hive, add those to `core-site.xml` to Hadoop configuration
+ directory on each node. For example, change `alluxio.user.file.writetype.default` from default `MUST_CACHE` to `CACHE_THROUGH`:
```

Comment:
Okay, which is the recommended way? I don't see we suggest using `alluxio-site.properties` in any other docs. So for consistency, I prefer the PR as is. And if we decide to recommend a different way, we should do it in all docs in another PR. What do you think?